src/jit/flowgraph.cpp

```diff
@@ -19810,23 +19810,40 @@ Compiler::fgWalkResult      Compiler::fgStress64RsltMulCB(GenTreePtr* pTree, fgW
     
     if (tree->gtOper != GT_MUL || tree->gtType != TYP_INT || (tree->gtOverflow())) {
         return WALK_CONTINUE;
-}
+    }
+
+#ifdef DEBUG
+    if (pComp->verbose)
+    {
+        printf("STRESS_64RSLT_MUL before:\n");              
+        pComp->gtDispTree(tree);
+    }
+#endif // DEBUG
 
     // To ensure optNarrowTree() doesn't fold back to the original tree.
```

Comment:
I believe that it's the insertion of the `GT_NOP` that ensures that `optNarrowTree()` doesn't fold - so perhaps that comment should remain just above the `GT_NOP`. In fact, it might be good to elaborate on that point.